Scripts/Modules/TooltipModule/TooltipModule.cs

```diff
@@ -39,6 +42,15 @@ class TooltipData
             public Material customHighlightMaterial;
             public bool persistent;
             public float duration;
+            public ITooltipPlacement placement;
+
+            public Transform GetTooltipTarget(ITooltip tooltip)
+            {
+                if (placement != null)
+                    return placement.tooltipTarget;
+
+                return ((MonoBehaviour)tooltip).transform;
```

Comment:
Is it safe to always assume this cast, before polling for transform?  What if something other than a MonoBehaviour implements ITooltip?

Reply:
This has come up once before. The problem is that otherwise we have literally no way of knowing where to position the tooltip. We could just go ahead and implement the default view-space tooltip now, but otherwise, a tooltip without placement that is not a MonoBehaviour is an invalid case and should throw an error.